setup.py

```diff
@@ -434,6 +440,21 @@ def get_tag(self):
     extra_compile_args = ['-Wno-unused-function']
 
 
+# For mac, ensure extensions are built for macos 10.9 when compiling on a
+# 10.9 system or above, overriding distuitls behaviour which is to target
+# the version that python was built for. This may be overridden by setting
+# MACOSX_DEPLOYMENT_TARGET before calling setup.py
+if is_platform_mac():
+    if 'MACOSX_DEPLOYMENT_TARGET' not in os.environ:
+        current_system = \
+            list(map(int, platform.mac_ver()[0].split('.')[:2]))
```

Comment:
Can you do a `LooseVersion` comparison here instead?

Reply:
also done. 

Incidentally, it seems the `distutils.version` isnt't documented, so may not be intended as a public API (?) I've come across suggestions to use `packaging.version` as an alternative for that reason.